### PR TITLE
update config reference to use lowercase values

### LINKED ISSUE #909: update docs references to use lowercase config values
### Docs Update Description

A combination of #876 and #871 updated `fidesops.toml` to use lowercase values. The docs should be updated to match.

### Additional context

A combination of merges made it so @ethyca/docs-authors wasn't tagged automatically.

### PR DESCRIPTION
**Pull request by conceptualshark**
# Purpose

The `fidesops.toml` file was updated to use lowercase values. The documentation has been changed to match.

# Checklist
- [x] Update [`CHANGELOG.md`](https://github.com/ethyca/fidesops/blob/main/CHANGELOG.md) file
  - [x] Merge in main so the most recent `CHANGELOG.md` file is being appended to
  - [x] Add description within the `Unreleased` section in an appropriate category. Add a new category from the list at the top of the file if the needed one isn't already there.
  - [x] Add a link to this PR at the end of the description with the PR number as the text. example: [#1](https://github.com/ethyca/fidesops/pull/1)
- [x] Applicable documentation updated (guides, quickstart, postman collections, tutorial, fidesdemo, [database diagram](https://github.com/ethyca/fidesops/blob/main/docs/fidesops/docs/development/update_erd_diagram.md).
- If docs updated (select one):
  - [x] documentation complete, or draft/outline provided (tag docs-team to complete/review on this branch)

# Ticket

Fixes #909

### CODE REVIEW

File: docs/fidesops/docs/saas_connectors/example_configs/hubspot.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Fidesops uses the following Hubspot endpoints to retrieve and delete Personally
 ## Connection Settings
 Fidesops provides a [Postman collection](../../postman/using_postman.md) for easily establishing connections to your third party applications. Additional connection instructions may be found in the [configuration guide](../saas_config.md).
 
-**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `MASKING_STRICT` variable in your `fidesops.toml` file is set to `FALSE`. 
+**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `masking_strict` variable in your `fidesops.toml` file is set to `False`.
```
Comment (sanders41) [inline docs/fidesops/docs/saas_connectors/example_configs/hubspot.md:21]:
```suggestion
**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `masking_strict` variable in your `fidesops.toml` file is set to `false`.
```

File: docs/fidesops/docs/saas_connectors/example_configs/mailchimp.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Fidesops uses the following Mailchimp endpoints to retrieve and delete Personall
 ## Connection Settings
 Fidesops provides a [Postman collection](../../postman/using_postman.md) for easily establishing connections to your third party applications. Additional connection instructions may be found in the [configuration guide](../saas_config.md).
 
-**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `MASKING_STRICT` variable in your `fidesops.toml` file is set to `FALSE`. 
+**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `masking_strict` variable in your `fidesops.toml` file is set to `False`.
```
Comment (sanders41) [inline docs/fidesops/docs/saas_connectors/example_configs/mailchimp.md:16]:
```suggestion
**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `masking_strict` variable in your `fidesops.toml` file is set to `false`.
```

File: docs/fidesops/docs/saas_connectors/example_configs/outreach.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ Fidesops uses the following Outreach endpoints to retrieve and delete Personally
 ## Connection Settings
 Fidesops provides a [Postman collection](../../postman/using_postman.md) for easily establishing connections to your third party applications. Additional connection instructions may be found in the [configuration guide](../saas_config.md).
 
-**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `MASKING_STRICT` variable in your `fidesops.toml` file is set to `FALSE`. 
+**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `masking_strict` variable in your `fidesops.toml` file is set to `False`.
```
Comment (sanders41) [inline docs/fidesops/docs/saas_connectors/example_configs/outreach.md:16]:
```suggestion
**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `masking_strict` variable in your `fidesops.toml` file is set to `false`.
```

File: docs/fidesops/docs/saas_connectors/example_configs/salesforce.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ For more information, see the [Salesforce sObject API reference](https://develop
 ## Connection Settings
 Fidesops provides a [Postman collection](../../postman/using_postman.md) for easily establishing connections to your third party applications. Additional connection instructions may be found in the [configuration guide](../saas_config.md).
 
-**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `MASKING_STRICT` variable in your `fidesops.toml` file is set to `FALSE`. 
+**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `masking_strict` variable in your `fidesops.toml` file is set to `False`.
```
Comment (sanders41) [inline docs/fidesops/docs/saas_connectors/example_configs/salesforce.md:21]:
```suggestion
**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `masking_strict` variable in your `fidesops.toml` file is set to `false`.
```

File: docs/fidesops/docs/saas_connectors/example_configs/sendgrid.md [hunk 5]
```diff
@@ -11,7 +11,7 @@ Fidesops uses the following Sendgrid endpoints to retrieve and delete Personally
 ## Connection Settings
 Fidesops provides a [Postman collection](../../postman/using_postman.md) for easily establishing connections to your third party applications. Additional connection instructions may be found in the [configuration guide](../saas_config.md).
 
-**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `MASKING_STRICT` variable in your `fidesops.toml` file is set to `FALSE`. 
+**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `masking_strict` variable in your `fidesops.toml` file is set to `FALSE`.
```
Comment (sanders41) [inline docs/fidesops/docs/saas_connectors/example_configs/sendgrid.md:14]:
```suggestion
**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `masking_strict` variable in your `fidesops.toml` file is set to `false`.
```

File: docs/fidesops/docs/saas_connectors/example_configs/sentry.md [hunk 6]
```diff
@@ -16,7 +16,7 @@ Fidesops uses the following Sentry endpoints to retrieve and delete Personally I
 ## Connection Settings
 Fidesops provides a [Postman collection](../../postman/using_postman.md) for easily establishing connections to your third party applications. Additional connection instructions may be found in the [configuration guide](../saas_config.md).
 
-**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `MASKING_STRICT` variable in your `fidesops.toml` file is set to `FALSE`. 
+**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `masking_strict` variable in your `fidesops.toml` file is set to `False`.
```
Comment (sanders41) [inline docs/fidesops/docs/saas_connectors/example_configs/sentry.md:19]:
```suggestion
**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `masking_strict` variable in your `fidesops.toml` file is set to `false`.
```

File: docs/fidesops/docs/saas_connectors/example_configs/stripe.md [hunk 7]
```diff
@@ -23,7 +23,7 @@ Fidesops uses the following Stripe endpoints to retrieve and delete Personally I
 ## Connection Settings
 Fidesops provides a [Postman collection](../../postman/using_postman.md) for easily establishing connections to your third party applications. Additional connection instructions may be found in the [configuration guide](../saas_config.md).
 
-**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `MASKING_STRICT` variable in your `fidesops.toml` file is set to `FALSE`. 
+**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `masking_strict` variable in your `fidesops.toml` file is set to `False`.
```
Comment (sanders41) [inline docs/fidesops/docs/saas_connectors/example_configs/stripe.md:26]:
```suggestion
**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `masking_strict` variable in your `fidesops.toml` file is set to `false`.
```

File: docs/fidesops/docs/saas_connectors/example_configs/zendesk.md [hunk 8]
```diff
@@ -14,7 +14,7 @@ Fidesops uses the following Zendesk endpoints to retrieve and delete Personally
 ## Connection Settings
 Fidesops provides a [Postman collection](../../postman/using_postman.md) for easily establishing connections to your third party applications. Additional connection instructions may be found in the [configuration guide](../saas_config.md).
 
-**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `MASKING_STRICT` variable in your `fidesops.toml` file is set to `FALSE`. 
+**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `masking_strict` variable in your `fidesops.toml` file is set to `False`.
```
Comment (sanders41) [inline docs/fidesops/docs/saas_connectors/example_configs/zendesk.md:17]:
```suggestion
**Deletion requests** are fulfilled by masking PII via `UPDATE` endpoints. To [give fidesops permission](../../guides/configuration_reference.md#configuration-variable-reference) to remove PII using `DELETE` endpoints, ensure the `masking_strict` variable in your `fidesops.toml` file is set to `false`.
```

**Review by sanders41 — CHANGES_REQUESTED** [review]

File: docs/fidesops/docs/guides/configuration_reference.md [hunk 9]
```diff
@@ -20,91 +20,92 @@ The `fidesops.toml` file should specify the following variables:
...
-|`CELERY_CONFIG_PATH` | `FIDESOPS__EXECUTION__CELERY_CONFIG_PATH` | string | data/config/celery.toml | N/A | An optional override for the [Celery](#celery-configuration) configuration file path.
-|`WORKER_ENABLED` | `FIDESOPS__EXECUTION__WORKER_ENABLED` | bool | True | True | By default, fidesops uses a dedicated [Celery worker](#celery-configuration) to process privacy requests asynchronously. Setting `WORKER_ENABLED` to `False` will run the worker on the same node as the webserver.
+|`privacy_request_delay_timeout` | `FIDESOPS__EXECUTION__PRIVACY_REQUEST_DELAY_TIMEOUT` | int | 3600 | 3600 | The amount of time to wait for actions delaying privacy requests, for example pre and post processing webhooks.
+|`task_retry_timeout` | `FIDESOPS__EXECUTION__TASK_RETRY_COUNT` | int | 5 | 0 | The number of times a failed request will be retried
```
[the remark this answers is not in the document]
Comment (conceptualshark) [reply docs/fidesops/docs/guides/configuration_reference.md:49]: 🦅 good eye on this one, thank you!

**Review by sanders41 — APPROVED** [review]

**Pull request merged**